### DISCUSSION TITLE
array index in spinful fermion basis

### DISCUSSION THREAD

**Discussion by MoorishQubit** [QuSpin/QuSpin#484] [Q&A]
Hi,

I am working with the fermi hubbard model in quspin. how do i change the array index of the spinful_fermion_basis_1d to match the computational basis ? 
![Screenshot 2021-11-04 at 11-21-51 Fermi-Hubbard - Jupyter Notebook](https://user-images.githubusercontent.com/36313185/140297477-3ab7f851-8d93-4d36-b144-f2addf4ee3ae.png)
the picture above is what quspin gives for 1 site. but what i want is |0>|0> to have index 0,  |0>|1> to have index 1 ... ect

Thanks,

**Comment by mgbukov** [comment]
This is a convention and I don't think you can do this in quspin; What you *can* do though is cast the hamiltonian object to a numpy/scipy array, and reverse your matrix once it's constructed, cf `hamiltonian.to_array()`, etc. functions.  

Also, take a look at this [example](https://weinbe58.github.io/QuSpin/examples/example00.html#example-00-label), it may help understand this notation.
